### PR TITLE
[10.0][ADD] shopinvader_cart_expiry: add new feature to manage cart expiry delay

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
LGTM (code only)

File: shopinvader_cart_expiry/views/shopinvader_backend.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<odoo>
+    <record model="ir.ui.view" id="shopinvader_backend_form_view">
+        <field name="name">shopinvader.backend.form (in shopinvader_cart_expiry)</field>
+        <field name="model">shopinvader.backend</field>
+        <field name="inherit_id" ref="shopinvader.shopinvader_backend_view_form"/>
+        <field name="priority" eval="90"/>
+        <field name="arch" type="xml">
+            <page name="sale" position="inside">
+                <group>
+                    <group string="Cart Expiry Management">
```
Comment (simahawk) [inline shopinvader_cart_expiry/views/shopinvader_backend.xml]: better to add a name to the group to make it compat w/ v12. Not a blocker

**Review by simahawk — APPROVED** [review]
LG

File: shopinvader_cart_expiry/views/shopinvader_backend.xml (hunk 1 above)
Comment (Cedric-Pigeon) [reply shopinvader_cart_expiry/views/shopinvader_backend.xml]: @simahawk  You are right. Just did it. Thanks

**Comment by Cedric-Pigeon** [comment]
@lmignon great day to merge, isn't it?

**Review by lmignon — APPROVED** [review]

**Pull request merged**